db/migrate/0004_add_position_to_issue_templates.rb

```diff
@@ -1,13 +1,16 @@
-class AddPositionToIssueTemplates < ActiveRecord::Migration
-  def self.up
-    add_column :issue_templates, :position, :integer, :default => 1
-
-    IssueTemplate.reset_column_information
-    issue_templates = IssueTemplate.all
-    issue_templates.each_with_index {|t, i| t.update_attribute(:position, i+1)}
-  end
-
-  def self.down
-    remove_column :issue_templates, :position
-  end
-end
+class AddPositionToIssueTemplates < ActiveRecord::Migration
+  def self.up
+    add_column :issue_templates, :position, :integer, default: 1
+
+    IssueTemplate.reset_column_information
```

Comment:
__[rails_best_practices]__
* use say with time in migrations [:ref](http://rails-bestpractices.com/posts/2010/08/19/use-say-and-say_with_time-in-migrations-to-make-a-useful-migration-log/)